.env.production.example

```
@@ -0,0 +1,22 @@
+# PostgreSQL Configuration
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgresPassword
+POSTGRES_DB=backtrade
+
+# Cloudflare Tunnel
+TUNNEL_TOKEN=
+
+# Node Configuration
+NODE_ENV=development
+API_HOST=0.0.0.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_HOST key should go before the NODE_ENV key

(UnorderedKey)